.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,61 @@
+name: Feature Request
+description:  Create a feature request for new features
+labels: ["Type: Feature Request", "Needs: Triage", "Needs: Lead"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT: Before posting, remove sensitive data, such as passwords, secret keys, session cookies, etc.**
+
+        Thanks for showing interest in adding new features to openlibrary! 👍
+
+  - type: textarea
+    attributes:
+      label: Problem Description
+      description: Describe the problem that this feature will address
+      value: |
+        ### A clear and concise description of what you want to happen
+
+        ### Expected behaviour / screenshots (ex: Figma design screenshots for UI feature)
+
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Proposal & Constraints
+      description: Describe the proposed solution and possible constraints
+      value: |
+        ### What is the proposed solution / implementation?
+
+        ### Is there a precedent of this approach succeeding elsewhere?
+
+        ### Which suggestions or requirements should be considered for how feature needs to appear or be implemented?
+
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Additional Context
+      description: Additional information about the feature request.
+      value: |
+        ### Other context or screenshots about the feature request
+
+  - type: textarea
+    attributes:
+      label: Stakeholders
+      description: Tag stakeholders of this feqture request if any
+      placeholder: "@ tag stakeholders related to this feature request"
+
+  - type: dropdown
+    attributes:
+      label: Priority
+      description: Describe the priority on which this feature should be implemented
+      options:
+        - 1
+        - 2
+        - 3
+        - 4
+        - 5
+      default: 2
```

Comment:
```suggestion
```

I like the idea of having the priority in the form I'd propose we add a small question about it to the problem section. Something like "How urgent is this feature request for you?" The reason being that it might cause a little confusion with priority which staff assign as labels to GitHub issues.